src/main/java/in/org/projecteka/hiu/HiuConfiguration.java

```diff
@@ -15,21 +15,8 @@
 import in.org.projecteka.hiu.common.CentralRegistryTokenVerifier;
 import in.org.projecteka.hiu.common.UserAuthenticator;
 import in.org.projecteka.hiu.common.UserBase64Authenticator;
-import in.org.projecteka.hiu.consent.ConsentManagerClient;
-import in.org.projecteka.hiu.consent.ConsentRepository;
-import in.org.projecteka.hiu.consent.ConsentService;
-import in.org.projecteka.hiu.consent.DataFlowRequestPublisher;
-import in.org.projecteka.hiu.consent.HealthInformationPublisher;
-import in.org.projecteka.hiu.dataflow.DataAvailabilityPublisher;
-import in.org.projecteka.hiu.dataflow.DataFlowClient;
-import in.org.projecteka.hiu.dataflow.DataFlowRepository;
-import in.org.projecteka.hiu.dataflow.DataFlowRequestListener;
-import in.org.projecteka.hiu.dataflow.DataFlowService;
-import in.org.projecteka.hiu.dataflow.DataFlowServiceProperties;
-import in.org.projecteka.hiu.dataflow.Decryptor;
-import in.org.projecteka.hiu.dataflow.HealthInfoManager;
-import in.org.projecteka.hiu.dataflow.HealthInformationRepository;
-import in.org.projecteka.hiu.dataflow.LocalDataStore;
+import in.org.projecteka.hiu.consent.*;
+import in.org.projecteka.hiu.dataflow.*;
 import in.org.projecteka.hiu.dataprocessor.DataAvailabilityListener;
```

Comment:
We chose to not use wild card imports for future name collisions.

Reply:
changed it back